cmd/node/factory/structs.go

```diff
@@ -1725,7 +1721,7 @@ func newForkDetector(
 		return processSync.NewMetaForkDetector(rounder, headerBlackList, blockTracker, genesisTime)
 	}
 
-	return nil, ErrCreateForkDetector
+	return nil, errors.New("could not create fork detector")
```

Comment:
because in this file where a new error is needed it is created in the place where is returned . here is the only place where it is used a global error to return

Reply:
ok